File: pl-domain/municipality-service/src/main/java/com/polylab/municipality/MunicipalityServiceApplication.java
package com.polylab.municipality;

import com.jxareas.jxcore.annotations.EnableJxCore;
import com.jxareas.jxcore.annotations.EnableNonRestrictiveCorsPolicy;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableJxCore
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
public class MunicipalityServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(MunicipalityServiceApplication.class, args);
    }

}

File: pl-domain/municipality-service/src/main/java/com/polylab/municipality/MunicipalityServiceApplication.kt
package com.polylab.municipality;

import com.jxareas.efficax.annotations.EnableEfficax
import com.jxareas.efficax.annotations.EnableNonRestrictiveCorsPolicy
import com.jxareas.efficax.common.extensions.runApp
import org.springframework.boot.autoconfigure.SpringBootApplication

@EnableEfficax
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
open class MunicipalityServiceApplication

fun main(vararg args : String) = runApp(MunicipalityServiceApplication::class, *args)
